packages/greencheck/src/Sitecheck/Logger.php

```diff
@@ -16,9 +17,40 @@ class Logger
      */
     private $entityManager;
 
-    public function __construct(EntityManager $entityManager)
+    /**
+     * @var Client
+     */
+    public $redis;
+
+    /**
+     * @var config
+     */
+    private $config;
+
+    public function __construct(EntityManager $entityManager, $config)
     {
         $this->entityManager = $entityManager;
+        $this->config;
```

Comment:
i would prefer to directly pass in the Redis object and not do the configuration in this class. The logger doesn't and shouldn't need to care about the config of it's adapters, just pass in the adapter directly.